data/lib/src/datasource_impl/document_datasource_impl.dart

```diff
@@ -110,7 +111,7 @@ class DocumentDataSourceImpl implements DocumentDataSource {
   Future<DownloadTaskId> downloadDocument(DocumentId documentId, Token token, Uri baseUrl) async {
     var externalStorageDirPath;
     if (Platform.isAndroid) {
-      externalStorageDirPath = (await getExternalStorageDirectory()).absolute.path;
+      externalStorageDirPath = (await ExtStorage.getExternalStoragePublicDirectory(ExtStorage.DIRECTORY_DOWNLOADS));
```

Comment:
Do we need wrap `await ExtStorage.getExternalStoragePublicDirectory(ExtStorage.DIRECTORY_DOWNLOADS)` by `( )`?

Reply:
:( okie.